### PR TITLE
feat: 🎸 select portfolio on asset issuance

### PR DESCRIPTION
**Pull request by sansan**
### Description

Adds posibility to set portfolio to which the Asset will be issued when creating asset (if the initial supply gt 0) and when issuing tokens;

### JIRA Link

https://polymesh.atlassian.net/browse/DA-833

### Checklist

- [x] Updated the Readme.md (if required) ?

### CODE REVIEW

File: src/api/procedures/issueTokens.ts [hunk 1]
```diff
@@ -50,11 +51,15 @@ export async function prepareIssueTokens(
     });
   }
 
-  const defaultPortfolio = await signingIdentity.portfolios.getPortfolio();
+  console.log({ portfolioId: portfolioId?.toString() });
```
Comment (polymath-eric) [inline src/api/procedures/issueTokens.ts:54]:
```suggestion
```

File: src/api/procedures/issueTokens.ts [hunk 2]
```diff
@@ -50,11 +51,15 @@ export async function prepareIssueTokens(
...
+
+  const portfolio = portfolioId
```
Comment (polymath-eric) [inline src/api/procedures/issueTokens.ts:56]: I guess we could make this 5.4 compatible by also checking the `isV5` param, but imo its better to wait for us to release the dual version, clean up, and then merge this PR in vs offering an API that won't work as expected on 5.4

File: src/api/procedures/issueTokens.ts [hunk 3]
```diff
@@ -9,6 +9,7 @@ import { bigNumberToBalance, portfolioToPortfolioKind, stringToTicker } from '~/
 export interface IssueTokensParams {
   amount: BigNumber;
   ticker: string;
+  portfolioId?: BigNumber;
```
Comment (polymath-eric) [inline src/api/procedures/issueTokens.ts:12]: Do we have to worry about custodian portfolios? We might want to allow for a full portfolio ID here (`portfolio: {did: string; id: BigNumber}`).

You might need to check with core what the full intention is here.

Comment (polymath-eric) [reply src/api/procedures/issueTokens.ts:12]: seems like its OK inferring the DID here

**Review by polymath-eric — APPROVED** [review]
lgtm

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**